snapcraft/parts/yaml_utils.py

```diff
@@ -88,8 +88,8 @@ def load(filestream: TextIO) -> Dict[str, Any]:
 
         if build_base is None:
             raise errors.LegacyFallback("no base defined")
-        if build_base != "core22":
-            raise errors.LegacyFallback("base is not core22")
+        if build_base in ("core", "core18", "core20"):
```

Comment:
This might be worth putting into a constant somewhere, as I have a feeling we'll be using it more

Reply:
Done!